ServerCore/Pages/EventSpecific/FAQ.cshtml

```diff
@@ -5,6 +5,27 @@
     ViewData["AdminRoute"] = "/EventSpecific/FAQ";
     ViewData["AuthorRoute"] = "/EventSpecific/FAQ";
     ViewData["PlayRoute"] = "/EventSpecific/FAQ";
+    var eventPage = Model as ServerCore.ModelBases.EventSpecificPageModel;
+    var Event = eventPage?.Event;
 }
 
-<partial name="../Resources/@(Model.Event.HomePartial ?? "Default")/FAQPartial.cshtml" />
\ No newline at end of file
+@if ((Event.EventBegin.Year < 2024) || string.IsNullOrEmpty(Event.HomePartial))
+{
+    if (!string.IsNullOrEmpty(Event.FAQContent))
+    {
+        @Html.Raw(Event.FAQContent);
+    }
+    else
+    {
+        <partial name="../Resources/@(Event.HomePartial ?? "Default")/FAQPartial.cshtml" />
+    }
+}
+else
+{
+    <environment include="Development">
+        <iframe id="faq-frame" style="width:100%;height:100%;overflow-x:scroll;overflow-y:visible;" title="FAQ Content" sandbox="allow-same-origin allow-scripts allow-popups" src="http://127.0.0.1:10000/devstoreaccount1/evt@(Model.Event.ID)/resources/faq-content.html"></iframe>
+    </environment>
+    <environment exclude="Development">
+        <iframe id="faq-frame" style="width:100%;height:100%;overflow-x:scroll;overflow-y:visible;" title="FAQ Content" sandbox="allow-same-origin allow-scripts allow-popups" src="https://puzzleserverteststore.blob.core.windows.net/evt@(Model.Event.ID)/resources/faq-content.html"></iframe>
```

Comment:
I don't think I understand the sizing here. Doesn't this need some kind of resize.js logic to size properly?

Reply:
If we want it to look as nice as the embedded puzzle do, yes.  Initially I wasn't sure if we did though, so I can add that.  It does add a slight pre-requisite to the embedded files, which I was trying to avoid, but if you're fine with that, especially since I will cover the need for the pre-requisite in the wiki page I'm writing, then I have no problem doing it. 

The only question I have is:  are you okay with this living in the wwwroot folder for these pages to use since we don't have event-agnostic file management via Azure storage?  We actually already have a size.js file that's being included but doesn't appear to have anything in it that we actually use, so I'd just add the code there.